metrics/spearmanr/spearmanr.py

```diff
@@ -38,19 +37,39 @@
 
 _KWARGS_DESCRIPTION = """
 Args:
-    predictions: Predicted labels, as returned by a model.
-    references: Ground truth labels.
+    - **`predictions`** (`list` of `float`): Predicted labels, as returned by a model.
+    - **`references`** (`list` of `float`): Ground truth labels.
+    - **`return_pvalue`** (`bool`): If `True`, returns the p-value. If `False`, returns
+                                    only the spearmanr score. Defaults to `False`.
```

Comment:
You can use the same format as other metric docstrings:
```suggestion
    - predictions (`List[float]`): Predicted labels, as returned by a model.
    - references (`List[float]`): Ground truth labels.
    - return_pvalue (`bool`): If `True`, returns the p-value. If `False`, returns
            only the spearmanr score. Defaults to `False`.
```